README.md

```diff
@@ -30,43 +30,166 @@ Software required for the proper functioning of the RSAN will be deployed on the
-herculesteam/augeasproviders_core (>= 2.1.5 < 4.0.0)
-herculesteam/augeasproviders_shellvar (>= 1.2.0 < 5.0.0)
-puppetlabs/apt (>= 2.0.0 < 8.0.0)
-puppet-grafana (>= 3.0.0 < 7.0.0)
-puppet-telegraf (>= 2.0.0 < 4.0.0)
-puppetlabs-apt (>= 4.3.0 < 8.0.0)
-puppetlabs-inifile (>= 2.0.0 < 5.0.0)
-puppetlabs-puppetserver_gem (>= 1.1.1 < 3.0.0)
-puppet/openvpn (>= 8.3.0)
+#### Module Dependencies
 
+- derdanne/nfs (>= 2.1.5)
+- puppetlabs/postgresql (>= 6.6.0)
+- puppetlabs/puppet_metrics_dashboard (>= 2.3.0)
+- puppetlabs/stdlib (>= 4.5.0 < 7.0.0)
+- puppetlabs/concat (>= 1.1.2 < 7.0.0)
+- puppetlabs/transition (>= 0.1.0 < 1.0.0)
+- herculesteam/augeasproviders_core (>= 2.1.5 < 4.0.0)
+- herculesteam/augeasproviders_shellvar (>= 1.2.0 < 5.0.0)
+- puppetlabs/apt (>= 2.0.0 < 8.0.0)
+- puppet-grafana (>= 3.0.0 < 7.0.0)
+- puppet-telegraf (>= 2.0.0 < 4.0.0)
+- puppetlabs-apt (>= 4.3.0 < 8.0.0)
+- puppetlabs-inifile (>= 2.0.0 < 5.0.0)
+- puppetlabs-puppetserver_gem (>= 1.1.1 < 3.0.0)
 
 
+#### Minimum Hardware requirements
+
+
+| AWS EC2|Cores| RAM |Disk|
+| --- | ----------- | --| --|
+| m1.medium | 2 CPU | 4GB Memory | 40GB Disk 
+
+#### OS Restrictions
+
+RSAN will support RHEL / Debian / Ubuntu however due to the additional of PE Client tools in the installation, you are restricted to installing it on a platform with the same OS as the Primary PE Server.
 
 ### Beginning with rsan
 
-RSAN has Two Classes:
+RSAN has two main classes for use in the installation:
 
  - rsan::exporter - to be applied to all Puppet infrastructure agents - Console node group "PE Infrastructure Agent"
- - rsan::importer - to be applied to a single node which will be come the Remote Support Access Node
+ - rsan::importer - to be applied to a single node which will be come the Remote Support Access Node(RSAN)
+
+Following the application of these clases to the infrastructure, Puppet Will need to be run on the corresponding agents in the following order:
 
-Adding these two classes will set up all applications and configurations to run  RSAN
+Infrastructure Agent(s)->RSAN Agent->Infrastrcture Agent(s)->RSAN Agent
 
 ## Usage
+The following outlines the main features of RSAN and how to consume them
+### Live Telemetry Display
+
+The Rsan node will host an instance of the [Puppet Metrics Dashboard](https://forge.puppet.com/modules/puppetlabs/puppet_metrics_dashboard)
+ 
+The Dashboard can be accessed on
+
+**URL:** http://<RSAN-ip\>:3000\
+**User:** admin\
+**Password:** admin
+
+For advanced configuration and documentation please see [Puppet Metrics Dashboard](https://forge.puppet.com/modules/puppetlabs/puppet_metrics_dashboard)
+
+### Infrastructure node file and log access	
+
+The RSAN node will, by default, mount `/var/log/`, `/opt/puppetlabs` and `/etc/puppetlabs` from each of the Puppet Enterprise Infrastructure nodes on the RSAN platform in the following location, as read-only file systems.
+
+`/var/pesupport/<FQDN of Infrastructure node\>/var/log\`
```

Comment:
My bad on these. They need to be an unordered list and remove the `\>` now that they are in the inline code.

```
* `/var/pesupport/<FQDN of Infrastructure node>/var/log`
```